AWS_FOR_FLUENT_BIT_VERSION

```diff
@@ -1 +1 @@
-2.16.0
+2.17.0
```

Comment:
This can be a patch release. Its just a fix for security issues. No new features. This is the standard case for patch bump.